Poker.BE/Poker.BE.Domain/Game/Hand.cs

```diff
@@ -167,43 +184,53 @@ private void PlaceAntes()
         private void PlaceSmallBlind()
         {
             if (CurrentRound.CurrentPlayer.Wallet.AmountOfMoney < gameConfig.MinimumBet / 2)
-                CurrentRound.PlayMove(Round.Move.Allin, gameConfig.MinimumBet / 2);
+                CurrentRound.PlayMove(Round.Move.Allin, 0);
             else
                 CurrentRound.PlayMove(Round.Move.Raise, gameConfig.MinimumBet / 2);
         }
 
         private void PlaceBigBlind()
         {
             if (CurrentRound.CurrentPlayer.Wallet.AmountOfMoney < gameConfig.MinimumBet)
-                CurrentRound.PlayMove(Round.Move.Allin, gameConfig.MinimumBet);
+                CurrentRound.PlayMove(Round.Move.Allin, 0);
             else
-                CurrentRound.PlayMove(Round.Move.Raise, gameConfig.MinimumBet);
+                //the raise amount is another small blind
+                CurrentRound.PlayMove(Round.Move.Raise, gameConfig.MinimumBet / 2);
+        }
+
+        private void UpdatePlayersBets()
+        {
+            foreach (Player player in CurrentRound.ActiveUnfoldedPlayers)
+            {
+                PlayersBets[player] += CurrentRound.LiveBets[player];
+            }
         }
 
         private void Showdown()
         {
-            throw new NotImplementedException();
+            //TODO
```

Comment:
❗️ we didn't finish it yet? make sure its synced with trello card. so we don't put it at *Complete*